runtime-js/numbers.js

```diff
@@ -216,6 +216,8 @@ atr$(JSNum$proto, 'character', function(){
   }
   return Character(this.valueOf());
 },undefined,function(){return{an:function(){return[shared(),actual()]},mod:$CCMM$,$cont:Integer,d:['$','Integer','$at','character']};});
+atr$(JSNum$proto, 'nearestFloat', function(){ return Float(this.valueOf()); },
+  undefined,function(){return{an:function(){return[shared(),actual()]},mod:$CCMM$,$cont:Integer,d:['$','Integer','$at','nearestFloat']};});
```

Comment:
This seems to work this way (at least the test suite runs now, previously it stopped with a `TypeError: Cannot call method 'equals' of undefined`.) Should I instead try to do what Gavin proposed?

> Add a file named  `nearestFloat.js`  in  `runtime-js/Integer`